iaso/models/base.py

```diff
@@ -819,15 +813,18 @@ def for_org_unit_hierarchy(self, org_unit):
 
     def filter_for_user(self, user):
         profile = user.iaso_profile
+        # Do a relative import to avoid an import loop
         from .org_unit import OrgUnit
```

Comment:
Good for me this time, but I think if we end up with too many local imports because of loops, we might actually need to restructure (divide more) the models folder.

Reply:
totally agree